src/main/cpp/t04_both2.cpp

```diff
@@ -23,9 +23,28 @@
 
 #include "t04_both2.h"
 #include <iostream>
+#include <algorithm>
+#include <set>
 
 using namespace std;
 
 int t04_both2() {
-
+	set<int> t, t1;
+	int N, M, cc = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'cc' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4732275)